policyengine_it/parameters/gov/agenzia_delle_entrate/credits/low_income/eligibility.yaml

```diff
@@ -0,0 +1,10 @@
+description: Italy limits low income tax credit eligibility to individuals making less than or equal to this threshold per year, depending on employment status.
+metadata:
+  unit: currency-EUR
```

Comment:
we need to a breakdown metadata piece:
 e.g.:
   breakdown:
    - [variable_class_name]